cost-analyzer/templates/_helpers.tpl

```diff
@@ -72,238 +36,114 @@ Kubecost 2.0 preconditions
-          "serviceKeySecret": "{{ .Values.kubecostProductConfigs.awsServiceKeyPassword }}"
-          {{- end }}
-      }
-    ]
-    {{- end }}
-  }
-{{- end }}
-
-{{/*
-Cloud integration source contents check. Either the Secret must be specified or the JSON, not both.
-Additionally, for upgrade protection, certain individual values populated under the kubecostProductConfigs map, if found,
-will result in failure. Users are asked to select one of the two presently-available sources for cloud integration information.
-*/}}
-{{- define "cloudIntegrationSourceCheck" -}}
-  {{- if and (.Values.kubecostProductConfigs).cloudIntegrationSecret (.Values.kubecostProductConfigs).cloudIntegrationJSON -}}
-    {{- fail "\nkubecostProductConfigs.cloudIntegrationSecret and kubecostProductConfigs.cloudIntegrationJSON are mutually exclusive. Please specify only one." -}}
-  {{- end -}}
-  {{- if and (.Values.kubecostProductConfigs).cloudIntegrationSecret ((.Values.kubecostProductConfigs).athenaBucketName) }}
-    {{- fail "\nkubecostProductConfigs.cloudIntegrationSecret and kubecostProductConfigs.athena* values are mutually exclusive. Please specifiy only one." -}}
-  {{- end -}}
-{{- if and (.Values.kubecostProductConfigs).cloudIntegrationJSON ((.Values.kubecostProductConfigs).athenaBucketName) }}
-    {{- fail "\nkubecostProductConfigs.cloudIntegrationJSON and kubecostProductConfigs.athena* values are mutually exclusive. Please specifiy only one." -}}
-  {{- end -}}
 {{- end -}}
 
 {{/*
 RBAC exclusivity check: make sure either simple RBAC or RBAC Teams is configured, not both
 */}}
-{{- define "rbacCheck" -}}
+{{- define "kubecost.rbac.check" -}}
   {{- if and (or (.Values.saml).groups (.Values.oidc).groups) (.Values.teams).teamsConfig  -}}
     {{- fail "\nSimple RBAC and RBAC Teams are mutually exclusive. Please specify only one." -}}
   {{- end -}}
 {{- end -}}
 
-{{/*
-Federated Storage source contents check. Either the Secret must be specified or the JSON, not both.
-*/}}
-{{- define "federatedStorageSourceCheck" -}}
-  {{- if and (.Values.kubecostModel).federatedStorageConfigSecret (.Values.kubecostModel).federatedStorageConfig -}}
-    {{- fail "\nkubecostModel.federatedStorageConfigSecret and kubecostModel.federatedStorageConfig are mutually exclusive. Please specify only one." -}}
-  {{- end -}}
-{{- end -}}
 
-{{/*
-Actions Storage source contents check. Either the Secret must be specified or the YAML, not both.
-*/}}
-{{- define "actionsStorageSourceCheck" -}}
-  {{- if ((.Values.kubecostProductConfigs).actions).enabled -}}
-  {{- if and ((.Values.kubecostProductConfigs).actions).storageConfigSecret ((.Values.kubecostProductConfigs).actions).storageConfig -}}
-    {{- fail "\nkubecostProductConfigs.actions.storageConfigSecret and kubecostProductConfigs.actions.storageConfig are mutually exclusive. Please specify only one." -}}
-  {{- end -}}
-  {{- end -}}
-{{- end -}}
```

Comment:
@Sean-Holcomb This is new code that @biancaburtoiu has introduced for our 3.0 MultiCluster Actions feature. Can we please keep this function?